DOC-600 Field level changes for release v.4.31 to v.4.37.0 (#65)
redpanda-data/rp-connect-docs: modules/components/pages/inputs/nats.adoc

== Authentication

> There are several components within Redpanda Connect which use NATS services. You will find that each of these components
> support optional advanced authentication parameters for https://docs.nats.io/nats-server/configuration/securing_nats/auth_intro/nkey_auth[NKeys^]
> and https://docs.nats.io/using-nats/developer/connecting/creds[User Credentials^].

Reviewer: ```suggestion
Several components in Redpanda Connect use NATS services. Each of these components
supports optional advanced authentication parameters for https://docs.nats.io/nats-server/configuration/securing_nats/auth_intro/nkey_auth[NKeys^]
and https://docs.nats.io/using-nats/developer/connecting/creds[User Credentials^].
```

Author: Updated as requested.
